src/main/java/harmonised/pmmo/setup/CommonSetup.java

```diff
@@ -52,7 +52,7 @@ public static void init(final FMLCommonSetupEvent event) {
 		
 		event.enqueueWork(() -> CriteriaTriggers.register("pmmo:skill_up", SkillUpTrigger.SKILL_UP));
 		//=========COMPAT=============
-		CurioCompat.hasCurio = ModList.get().isLoaded("curios");
```

Comment:
same with this.  it doesn't break anything to be uncommented, so might as well leave it so it isn't forgotten when compat is readded.